templates/displayApplicants.phtml

```diff
@@ -10,15 +10,23 @@
     <link rel="stylesheet" type="text/css" href="./css/genericStyles.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+    <script src="./js/populateDropdowns.js" defer></script>
     <script defer src="./js/applicantModal.js"></script>
     <script defer src="./js/deleteApplicant.js"></script>
+    <script defer src="./js/displayApplicants.js"></script>
 </head>
 <body>
 <div class="container">
     <div class="row">
         <div class="col-xs-12 registeredApplicants">
             <h1>Registered Applicants</h1>
             <form class="arrowForm" method="get">
+				<label for="cohorts">
+					Select Cohort
+					<select id="cohorts" class="submitApplicant form-control" name="cohortId">
+						<option value="all">All</option>
+					</select>
+				</label>
```

Comment:
indentation seems a bit off here? unless this is the correct way?

Reply:
I think this is GitHub being funny, looks fine in PHPStorm.